src/lib/UIComponent.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react'
 import renderComponent, { IRenderResultConfig } from './renderComponent'
+import { AccessibilityActions } from './accessibility/interfaces'
 
 class UIComponent<P, S> extends React.Component<P, S> {
   private readonly childClass = this.constructor as typeof UIComponent
   static defaultProps: { [key: string]: any }
   static displayName: string
   static className: string
   static handledProps: any
+  protected actions: AccessibilityActions
```

Comment:
I would suggest to make this name to be a bit more precise in terms of semantics. Here we are not defining the actions, but rather defining action handlers - the logic that should be performed by the component if specific action (or 'intent') has been raised by behavior. So, would suggest to rename to `actionHandlers`. This will also make it easy to read all the code snippets where this prop is used

Reply:
I'm having a bit different point of view, you might disagree :)
A component has actions - a map of things it can do. They can be raised or not, based on behavior now. The behavior could even don't exist. But the intent ```Hey, I am component and can do next things which you can call``` stays. In some time we can change our mind, and replace behaviors with something else, but still component will have actions which can be called directly or indirectly - by adding it to keyboard handlers.